nautobot/extras/tests/test_jobs.py

```diff
@@ -35,9 +35,147 @@
 from nautobot.extras.models.models import JobLogEntry
 
 
-class JobTest(TransactionTestCase):
+class JobTest(TestCase):
     """
-    Test basic jobs to ensure importing works.
+    Test job features that don't require a transaction test case.
```

Comment:
How can we reasonably determine which tests do or do not require transaction test cases?

Reply:
I left anything that uses `create_job_result_and_run_job` in a transaction test case